src/createServer.js

```diff
@@ -1,11 +1,190 @@
 'use strict';
 
 // const express = require('express');
+const express = require('express');
+
+const { Expenses } = require('./services/Expenses.js');
+const { Users } = require('./services/Users.js');
 
 function createServer() {
   // Use express to create a server
   // Add a routes to the server
   // Return the server (express app)
+  const app = express();
+
+  const expenses = new Expenses();
+  const users = new Users();
+
+  app.use(express.json());
+
+  app.get('/users', (req, res) => {
+    res.send(users.getUsers());
+  });
+
+  app.get('/users/:userId', (req, res) => {
```

Comment:
It's gonna be better to use Routes with prefixes, so we don't gonna have duplicates in endpoints like `/users` or `/expenses`